app/about_us/OurCrew/OurCrew.js

```diff
@@ -0,0 +1,66 @@
+import styles from "./OueCrew.module.css";
+
+export default function OurCrew() {
+  // TASK - React 1 week 1
+  // Create the "Our Crew section"
+  // Use the descriptions provided in /app/about_us/README.md
+  // Use the pictures from /public/crew
+  // Some inspiration ideas found in /data/inspiration_about_us
+  return (
+    <>
+      <div className={styles.grid}>
```

Comment:
We can just use the `<div>` as the root html node here:
```
return (
  <div>
    ...
  </div>
)
```

So you can get rid of the empty brackets `<>`

Reply:
I think it was for something, but when i delete it nothing hapepened, so i delete it